common/src/Microsoft.Azure.IIoT.Core/src/Diagnostics/Extensions/SerilogEx.cs

```diff
@@ -66,6 +66,6 @@ public static class SerilogEx {
         }
 
         private const string kDefaultTemplate =
-            "[{Timestamp:HH:mm:ss} {Level:u3} {SourceContext:lj}] {Message:lj} {NewLine}{Exception}";
+            "[{Timestamp:yyyy-MM-dd HH:mm:ss} {Level:u3} {SourceContext:lj}] {Message:lj} {NewLine}{Exception}";
```

Comment:
@monikavar, can the time zone also be added ? UTC would be preferable.

Reply:
Good point!
What would be more preferable? UTC time or local time with timezone?
e.g.,
1. `2020-09-07 14:22:29Z`
2. `2020-09-07 16:22:29 +02:00`

Reply:
I would prefer UTC one.

Reply:
done.